admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -1004,6 +1004,28 @@ class AdminModuleController {
   isReadMoreModalOpened() {
     return $('.modal-read-more').is(':visible');
   }
+
+  updateModuleVisibilityFromHash() {
+    const self = this;
+
+    if (window.location.hash !== '') {
+      const categoryRef = window.location.hash.substring(1);
+      self.currentCategoryFilter = categoryRef ? String(categoryRef).toLowerCase() : null;
+
+      const jqoCategory = $(`${self.categoryItemSelector}[data-category-ref="${self.currentCategoryFilter}"]`);
+
+      if (jqoCategory.length === 1) {
+        // Change dropdown label to set it to the current category's displayname
+        $(self.categorySelectorLabelSelector).text(jqoCategory.data('category-display-name'));
+        $(self.categoryResetBtnSelector).show();
+        self.updateModuleVisibility();
+      } else {
+        self.updateModuleVisibility();
+      }
+    } else {
+      self.updateModuleVisibility();
+    }
```

Comment:
What about

```suggestion
      }
    }
    self.updateModuleVisibility();
```

?

The method will always be called from my understanding.

Reply:
Oh my god. Thanks for review.

Reply:
Fixed